features/steps/steps.py

```diff
@@ -34,6 +36,11 @@ def get_inst_attributes(dc):
         yield 'inst_type', dc.inst.is_a()
         yield 'inst_id', dc.inst.id()
 
+def random_string():
```

Comment:
What's the purpose of this?

Reply:
I think it's there by accident with going back and forth to ALB002 for testing with pyparsing. Deleted it

Reply:
I was wrong, it was added in this commit https://github.com/buildingSMART/ifc-gherkin-rules/pull/21/commits/cca7b3ae03ac411c267c96efaa1b49e189685876  but unused due to some changes later on.